src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,43 +1,40 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.list.IntLinkedList;
+import edu.kis.vh.nursery.storage.IntLinkedList;
+import edu.kis.vh.nursery.storage.IntStorageInterface;
 
 public class DefaultCountingOutRhymer {
 
-    private IntArrayStack intArrayStack = new IntArrayStack();
-    private IntLinkedList intLinkedList = new IntLinkedList();
+    IntStorageInterface intStorageInterface = new IntLinkedList();
```

Comment:
"interface" suffix of the attribute name is wrong and does not show the meaning.
Should be initialized in the default constructor (inconsistency)